tests/tests/cpld_fv3_ccpp_384_mom6_cice_cmeps_35d

```diff
@@ -0,0 +1,82 @@
+#
+#  CPLD_FV3_CCPP_384_MOM6_CICE_CMEPS_35D_BMARK_RT test
+#
+
+export TEST_DESCR="Coupled FV3-CCPP-MOM6-CICE-CMEPS system"
+export_35d_run
+export_fv3
+export_cpl
+
+export SYEAR
+export SMONTH
+export SDAY="01"
+export SHOUR="00"
+
+export DAYS="35"
+export FHMAX="840"
+export FDIAG="6"
+export WLCLK=480
+
+export DT_ATMOS="450"
+export NPX="385"
+export NPY="385"
+export IMO="1536"
+export JMO="768"
+export FNALBC="'global_snowfree_albedo.bosu.t766.1536.768.rg.grb',"
+export FNVETC="'global_vegtype.igbp.t766.1536.768.rg.grb',"
+export FNSOTC="'global_soiltype.statsgo.t766.1536.768.rg.grb',"
+export FNSMCC="'global_soilmgldas.statsgo.t766.1536.768.grb',"
+export FNABSC="'global_mxsnoalb.uariz.t766.1536.768.rg.grb',"  
+
+export TASKS=866
+export TPN=40
+export INPES=6
+export JNPES=12
+export THRD=1
+export WRTTASK_PER_GROUP=24
+
+export med_petlist_bounds="0 431"
+export atm_petlist_bounds="0 455"
+export ocn_petlist_bounds="456 695"
+export ice_petlist_bounds="696 743"
```

Comment:
You'll want this to be the same as in your cold test, where you specify the cpl_bmark settings.

Reply:
in the 35d/cold with ww3, this is how it was as they are just copies of the two files edited to remove the WW3/wav variables/exports.
So the cold and 35d both need the dynamic variables and not hard coded values?
Should this also be true in the files with WW3?